### ISSUE TITLE
Rhythms + More TSL Examples

### ISSUE THREAD

**Issue by dc3347** [Barnard-PL-Labs/tsltools#23]
Creating more TSL to js examples

**Comment by dc3347** [comment]
Update 4/26/2022: 
This week I updated the [older blog examples](https://github.com/Barnard-PL-Labs/tsltools/commit/2a82eb8f7bf264c89593701ada2dea975812e8d5) with the new `if, else if` code that Mark wrote. I also made sure the examples still worked with the new code, fixing them in the [corresponding javascript](https://github.com/Barnard-PL-Labs/tsltools/commit/d4abc402bac8ebc5307c98a405e2ef60272462d1). I also began working on [adding rhythms](https://github.com/Barnard-PL-Labs/tsltools/commit/87f2f7e49388f5acbe647f073dbe048e0e2574bd) to two-note examples. Lastly, I added a section in the blog post for the [new rhythm example](https://github.com/Barnard-PL-Labs/tsltools/commit/dc13e1c9abcc70f5fe5a5442cb30c5fa0ba6948a) I created. 

From the email you sent, “I think eventually we should be generating an actual "step" function that can be called to step the mealy machine forward, rather than a block of code than then needs to be put into a function by the user. “ Could we get some clarification on this?
